Update of the meshModel (#687)
meshcloud/meshcloud-docs: docs/meshstack.index.md

![meshStack Context and Components](assets/meshstack-chart.svg)

> ##Central Concepts

Reviewer: ```suggestion
## Central Concepts
```
Space is required between header info & header text

> meshStack uses the concepts of Workspaces and Projects to represent the structure of existing application teams in the meshStack platform. Application Team can manage Tenants inside Projects. In order to reflect the organizations compliance and securiity standards in the cloud a Landing Zone is applied to every Tenant. Building Blocks can be leverage either as part of a mudular Landing Zone or additionally on top of a Tenant. Building Blocks are deployed based on the blueprint provided via the Building Block definition. 

Reviewer: ```suggestion
meshStack uses the concepts of Workspaces and Projects to represent the structure of existing application teams in the meshStack platform. Application Team can manage Tenants inside Projects. In order to reflect the organization's compliance and security standards in the cloud a Landing Zone is applied to every Tenant. Building Blocks can be leveraged either as part of a modular Landing Zone or additionally on top of a Tenant. Building Blocks are deployed based on the configuration provided in the Building Block definition. 
```

> meshStack supports a multi platform operation which means you can manage multiple platforms of the same or different typ with meshStack. Below you can see an overview of the relations between the individual elements of meshStack as objects.

Reviewer: ```suggestion
meshStack supports the management of multiple platforms of the same or different type. Below you can see an overview of the relations between the individual elements in meshStack as objects.
```

> ![meshModel](assets/meshModel.png)

Reviewer: So the problem seems you added a reference to the image but you didn't upload the image. Check your VSCode, it might be hanging around somewhere there.
